<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>778</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>425</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_open_barcodeport">
    <property name="geometry">
     <rect>
      <x>818</x>
      <y>30</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clear_result">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>743</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear result</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>168</y>
      <width>1072</width>
      <height>564</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="verticalSpacing">
      <number>3</number>
     </property>
     <item row="0" column="0">
      <widget class="QPlainTextEdit" name="logtextedit">
       <property name="minimumSize">
        <size>
         <width>1070</width>
         <height>340</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1070</width>
         <height>340</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Courier</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPlainTextEdit" name="logtextedit_barcode">
       <property name="minimumSize">
        <size>
         <width>1070</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1070</width>
         <height>90</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Courier</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPlainTextEdit" name="textedit_result">
       <property name="minimumSize">
        <size>
         <width>1070</width>
         <height>120</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1070</width>
         <height>120</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Courier</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>687</x>
      <y>9</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>110</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>110</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Barcode Port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clear_log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>743</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clear_barcodelog">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>743</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear barcode log</string>
    </property>
   </widget>
   <widget class="QLabel" name="msglabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>94</y>
      <width>1070</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1070</width>
      <height>68</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1070</width>
      <height>70</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Book Antiqua</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>988</x>
      <y>30</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QPushButton" name="startbutton">
    <property name="geometry">
     <rect>
      <x>391</x>
      <y>30</y>
      <width>140</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>140</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Test Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>110</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>110</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>WizFi630S Port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rescanbutton">
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>30</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ReScan COM ports&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>ReScan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_open_devport">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>30</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combobox_barcode">
    <property name="geometry">
     <rect>
      <x>688</x>
      <y>30</y>
      <width>115</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>115</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>115</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="combobox_devport">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>30</y>
      <width>115</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>115</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
